src/vivarium_public_health/dataset_manager/dataset_manager.py

```diff
@@ -105,16 +112,33 @@ def load(self, entity_key: str, keep_age_group_edges: bool=True, **column_filter
             if the data is a dataframe.
         """
         data = self.artifact.load(entity_key)
-        return filter_data(data, keep_age_group_edges, **column_filters) if isinstance(data, pd.DataFrame) else data
+        return filter_data(data, keep_age_group_edges, self.config_filter_term, **column_filters) if isinstance(data, pd.DataFrame) else data
```

Comment:
Is it possible for filter_data to return anything other than a DF?

Reply:
Filter data will break if it gets something that is not a dataframe and will always return a dataframe if it receives one.